### PR TITLE
Content edits deux

### PR DESCRIPTION
**Pull request by OpenGlobe**
Hey @emileighoutlaw - can you take a look?  I tried to integrate NTIS feedback in a few areas, and a bit from the content outline I was working on.  Feel free to cut / ignore anything here that you don't think works!

### CODE REVIEW

**Comment by emileighoutlaw** [comment]
Yeah — I'm going to take a look at this today. I've had some fires to put out for FEC, but I'm going to get to this later tonight. Everyone hang tight and resist the urge to merge 😁

File: index.html [hunk 1]
```diff
@@ -62,27 +62,27 @@
 
         <h3>How we work</h3>
         <p>
-          Government agencies produce extraordinary amounts of data, but they don&rsquo;t always know what to do with it.</p>
+          One of the government's largest untapped resources is data.  Agencies produce extraordinary amounts of data, but they don&rsquo;t always know what to do with it.</p>
```
Comment (emileighoutlaw) [inline index.html:65]: I just couldn't say that it was totally "untapped" because that's not true.

**Comment by emileighoutlaw** [comment]
Okay! This looks great, but I made some meaty changes, so I don't think I can merge. Perhaps it's back to you, @OpenGlobe? (I wouldn't mind a last look from your smart eyes on this)

File: index.html (hunk 1 above)
Comment (OpenGlobe) [reply index.html:65]: +1

**Comment by OpenGlobe** [comment]
Merged @emileighoutlaw !  Thanks for the review!

**Pull request merged**